data/data/agent/systemd/units/agent-interactive-console.service

```diff
@@ -2,6 +2,8 @@
 Description=Get interactive user configuration at boot
 After=network-pre.target NetworkManager.service pre-network-manager-config.service selinux.service
 Before=getty@tty1.service network.target network.service agent.service node-zero.service NetworkManager-wait-online.service
+ConditionPathExists=/usr/local/bin/agent-tui
+ConditionPathExists=!/etc/assisted/node0
 
 [Service]
 Type=oneshot
```

Comment:
Hi, I just hit this while testing the agent on arm64. Shouldn't `RemainAfterExit=yes` be also considered here?

Reply:
`RemainAfterExit=yes` is used in example here https://www.redhat.com/sysadmin/systemd-oneshot-service

Reply:
> I don't know the exact cause of it appearing twice,

in my case, continuously restarting without explicit errors after the users quit or after waiting for the non-action-needed timeout. I couldn't debug further as neither ssh nor TTYs were available at that stage.

Reply:
I suspect that the problem was related to the path and the burst limits/freq, even though the builds didn't show it up until one day ago, so there could some additional enviromental condition changed.
In any case, given the consolidated approach we adopted to make available the agent-tui binary, the systemd path is not effectively required anymore. The additional condition on node0 should prevent any further restarts, thus the `RemainAfterExit=yes` could be worth to be used, adding it.
Note that [last job failed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_installer/6977/pull-ci-openshift-installer-master-e2e-agent-compact-ipv4/1636084652940726272/artifacts/e2e-agent-compact-ipv4/baremetalds-devscripts-setup/artifacts/root/dev-scripts/logs/06_agent_create_cluster-2023-03-15-205617.log) but due a later problem in the installation, the initial boot phase was ok